Library/BookCopy.aspx.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Library.Data;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace Library
+{
+    public partial class BookCopy : System.Web.UI.Page
+    {
+        int bookID = 0;
+        int library_id = 0;
```

Comment:
Establish a convention for naming class fields and stick with it. In general I feel like your naming is consistent, but keep an eye out for inconsistencies like this.

Reply:
Looking further down in the `BookCopy` class, it looks like the `library_id` field can be turned into a variable that's local to the `Add_Click` method.